src/endpoints/users.js

```diff
@@ -30,6 +43,7 @@ export const getWhoami = contentTypesProvided([
     res.json({
       user: req.user || null,
       visibleGroups: visibleGroups(req.user),
+      groupMemberships: groupMemberships(req.user),
```

Comment:
_non-blocking_

I'm hesitant to start exposing all the basic properties of the Group class this way, e.g.:

```json
{
  "name": "test",
  "isPublic": true,
  "membershipRoles": {
    "viewers": "test/viewers",
    "editors": "test/editors",
    "owners": "test/owners"
  }
}
```

It seems fine at the moment (although `membershipRoles` is really internal and shouldn't be needed by the frontend; any need is a misuse). But I'm worried we won't realize we're automatically and publicly serializing the Group class some time down the road and we'll leak something we shouldn't.

Reply:
Good point. Addressed this with bd882f9503422fb7545e86ba72fbe8696947f988